button.py

```diff
@@ -0,0 +1,47 @@
+from constants import *
+
+class Button(pygame.sprite.Sprite):
+    def __init__(self, button_text, center_x, center_y):
+        super().__init__()
+        self.value = button_text
+        self.clicked = False
+        self.button_color = 'white'
+        self.shadow_color = CLAIRVOYANT
+
+        #create transparent button surface
+        self.image = pygame.Surface((250, 60), pygame.SRCALPHA)
+        self.rect = self.image.get_rect(centerx=center_x, centery=center_y)
+
+    def update(self):
+        #"clear" button image of previously draw elements
+        self.image.fill(pygame.Color(0,0,0,0))
+
+        #set button and shadow rectangles with correct positioning
+        self.shadow_rect = pygame.Rect(3, 5, 247, 55)
```

Comment:
If I understand you correctly, the position numbers you're referring to are the 3 and 5?  The button surface is 250 x 60, so I want the shadow and button rectangles to be a bit smaller to see the difference.  Hence, 247 x 55.  The button topleft corner is located at (0, 0), so the offset the shadow to start at (3, 5) so that we can see the shadow part below the button.

Reply:
Ok cool, got it now!